schemas/oic.r.alertcollection.json

```diff
@@ -0,0 +1,44 @@
+{
+  "id": "http://openconnectivity.org/iotdatamodels/schemas/oic.r.alertcollection.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description" : "Copyright (c) 2018 Open Connectivity Foundation, Inc. All rights reserved.",
+  "title": "Alert Collection",
+  "definitions": {
+    "oic.r.alertcollection": {
+      "properties": {
+        "rt": {
+          "type": "array",
+          "minItems": 1,
+          "maxItems": 1,
+          "uniqueItems": true,
+          "items": {
+            "enum": ["oic.r.alertcollection"]
+          }
+        },
+        "rts": {
```

Comment:
Do we need to say anywhere that "rts" (and "rt" and "links"??) are required Properties so that the following shall statement can be caught via schema validation:
"However the Alert Collection does impose restrictions of the values that shall be populated in the “rt” and “rts” Properties."

Reply:
"rt" and "rts" have been added as required Properties in the schema.